packages/app/src/components/inspector/TextStyleRow.tsx

```diff
@@ -0,0 +1,171 @@
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+import {
+  Label,
+  Select,
+  InputField,
+  LabeledElementView,
+} from 'noya-designsystem';
+import { Spacer } from 'noya-designsystem';
+import { useCallback, memo, useMemo } from 'react';
+import ColorInputFieldWithPicker from './ColorInputFieldWithPicker';
+import * as InspectorPrimitives from './InspectorPrimitives';
+
+interface TextStyleRowProps {
+  fontSize: number;
+  fontFamily: string;
+  fontColor: Sketch.Color;
+  lineSpacing: number;
+  letterSpacing: number;
+  paragraphSpacing: number;
+  onChangeFontSize: (value: number) => void;
+  onChangeFontFamily: (value: string) => void;
+  onChangeFontWeight: (value: string) => void;
+  onChangeFontColor: (color: Sketch.Color) => void;
+  onChangeLineSpacing: (value: number) => void;
+  onChangeLetterSpacing: (value: number) => void;
+  onChagenParagraphSpacing: (value: number) => void;
+}
+
+export default memo(function TextStyleRow({
+  fontColor,
+  fontSize,
+  fontFamily,
+  lineSpacing,
+  letterSpacing,
+  paragraphSpacing,
+  onChangeFontColor,
+  onChangeFontSize,
+  onChangeFontFamily,
+  onChangeFontWeight,
+  onChangeLineSpacing,
+  onChangeLetterSpacing,
+  onChagenParagraphSpacing,
+}: TextStyleRowProps) {
+  const characterInputId = `char`;
+  const lineInputId = `line`;
+  const paragraphInputId = `paragraph`;
+
+  const [family, size] = useMemo(
+    () => fontFamily.replace('MT', '').split('-'),
```

Comment:
FYI, we'll have to get font weight (light, bold, etc) from the font file itself I think (the .ttf or .otf). Using the name isn't reliable. We don't have a way to do that yet though, so you can leave this for now.

Reply:
Yeah, I did noticed that the MT wasn't in all the files 😅 But I also have all the fonts in an array, I'm guessing this will be handle better later. It's good to keep in mind.